content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+---
+title: Operators
+hasparent: true
+weight:
+---
+
+# Understanding Operators
+
+The Jaeger Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/). It provides a method of packaging, deploying, and managing Jaeger.
+
+Operators are pieces of software that ease the operational complexity of running another piece of software. More technically, _Operators_ are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an app that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` (kubernetes) or `oc` (OKD) tooling. To be able to make the most of Kubernetes, you need a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
```

Comment:
`s/app/application/`? I tend to think that "apps" are things that run on my phone, but it might be just me :)

Reply:
Nope, not just you.  I almost changed that when I was going through the content, but decided to leave it in.  Fixed now.